pkg/manifests/template/helm.go

```diff
@@ -170,19 +178,75 @@ func (h *helm) Render(svc *console.ServiceDeploymentForAgent, utilFactory util.F
 	return manifests, nil
 }
 
+func (h *helm) luaValues(svc *console.ServiceDeploymentForAgent) (map[string]interface{}, error) {
+	// Initialize empty results
+	newValues := make(map[string]interface{})
+	var valuesFiles []string
+
+	if svc == nil {
+		return nil, fmt.Errorf("no service found")
+	}
+	if svc.Helm == nil {
```

Comment:
can we compress these to just `svc.Helm == nil || svc.Helm.LuaScript == nil`?

Reply:
done